### PR TITLE
STCLI-109: Dynamically include CLI as devDependency of app and workspace

### PR DESCRIPTION
**Pull request by MaksymDryha**
According to [STCLI-109](https://issues.folio.org/browse/STCLI-109), CLI should be included as a devDependency of the app and workspace's package.json

### CODE REVIEW

File: lib/create-app.js [hunk 1]
```diff
@@ -2,14 +2,15 @@ const copy = require('kopy');
 const path = require('path');
 const kebabCase = require('just-kebab-case');
 const pascalCase = require('just-pascal-case');
+const cliPackageJSON = require('../package.json');
```
Comment (Godlevskyi) [inline lib/create-app.js]: what for to import the whole package.json?
you can for example destructure the version, like:
`const { version } = require('../package.json');`

File: lib/environment/development.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ const yarn = require('../yarn');
 const { allModules, uiModules, platforms, toFolioName } = require('./inventory');
 const AliasService = require('../platform/alias-service');
 const logger = require('../cli/logger')();
+const cliPackageJSON = require('../../package.json');
```
Comment (Godlevskyi) [inline lib/environment/development.js]: Same, no need to require the whole package.json

File: lib/environment/development.js [hunk 3]
```diff
@@ -113,6 +114,8 @@ module.exports = class DevelopmentEnvironment {
 
     // Copy the workspace template
     const templateDir = path.join(__dirname, '..', '..', 'resources', 'workspace');
+    const zeroPatchCLIVersion = cliPackageJSON.version.replace(/\d*$/, '0');
```
Comment (Godlevskyi) [inline lib/environment/development.js]: Not sure that such regex is correct if they will use additional labels for pre-release and build metadata
E.g  1.2.3-alpha.1 
You can have a look at `coerce` method of semver package (https://docs.npmjs.com/misc/semver)

File: lib/create-app.js [hunk 4]
```diff
@@ -1,15 +1,19 @@
...
 function appDefaults(inputName, inputDescription) {
   let appName = kebabCase(inputName);
 
+  const { version: normilizedCLIVirsion } = semver.coerce(currentCLIVersion);
+  const zeroPatchCLIVersion = normilizedCLIVirsion.replace(/\d*$/, '0');
+
   if (appName.startsWith('ui-')) {
     appName = appName.replace('ui-', '');
   }
-
```
Comment (VictorSoroka1) [inline lib/create-app.js]: I do not see a reason for removing this empty line.

**Pull request merged**